lib/deprecation-cop-view.coffee

```diff
@@ -90,10 +81,10 @@ class DeprecationCopView extends ScrollView
 
   serialize: ->
     deserializer: @constructor.name
-    uri: @getUri()
+    uri: @getURI()
 
   handleGrimUpdated: =>
-    @refreshCallsButton.show()
+    @updateCalls()
```

Comment:
> Just to be on the safe side, perhaps this should be wrapped in a _.debounce call here with maybe a 1 second timeout so that any deprecations occurring while the view is open don't lock up Atom as the view is re-rendered.

Oh wow, that's an excellent observation! Now that you've pointed that out, this might have been one of the reasons for not making the view update automatically initially, but instead manually (using the buttons). I think a debounce should keep us out of trouble -- thanks for catching/suggesting that, I've added it in 884f1c9.

Also tried to think of situations where we'd want to debounce the `updateSelectors` call as well, but couldn't think of anything so didn't add it for any of those calls.